frontend/src/container/WorkspaceContent.jsx

```diff
@@ -407,12 +407,12 @@ class WorkspaceContent extends React.Component {
     }
   }
 
-  handleClickFolder = async folderId => {
+  handleClickFolder = async (folderId, e) => {
```

Comment:
a common practice in js when overriding event handler is to keep the event as first parameter.
Just invert the parameters order

Reply:
`handleClickFolder` was used in `handleClickCreateContent` to only open/close the folder and fetch its data. It was not consistent because it isn't the same click event. 
So in my latest changes (9723a91a6985c49a66c3843f99d1fbf854e6f5b7), I created a function which open/close a folder and fetch its data